libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -457,9 +486,26 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
         this._dataSource = dataSource;
     }
 
+    /**
+     * @hidden return count for matching suggestion with input text
+     * @returns number
+     */
+    private _getSuggestionsLength(): number {
+        let count = 0;
+        this.dropdownValues$.subscribe((suggestions) => {
```

Comment:
should we have to unsubscribe for dropdownValues$?

Reply:
fixed